chore: clarify single key unwrapping during schema inference (MINOR) (#6795)
confluentinc/ksql: docs/concepts/schemas.md

The following example statements show how to create streams and tables that have 
Avro-formatted data. If you want to use Protobuf- or JSON-formatted data,
substitute `PROTOBUF` or `JSON_SR` for `AVRO` in each statement.

!!! note
    ksqlDB handles the `JSON` and `JSON_SR` formats differently. `JSON_SR` reads and 
    registers new schemas with {{ site.sr }} as necessary, while `JSON` requires you
    to specify the schema. Data formatted with `JSON_SR` is not binary compatible with
    the `JSON` format.
    
!!! note
    When performing schema inference to infer value schemas, ksqlDB takes user-specified
    [single field (un)wrapping](../developer-guide/serialization.md#single-field-unwrapping) 
    into account. When performing schema inference on key schemas, the resulting key schema

>     will always be unwrapped. If your key is contained in an outer record or object, the

Reviewer: ```suggestion
    is always unwrapped. If your key is contained in an outer record or object, the

```

>     inferred key will be represented as a `STRUCT`. You can access individual fields within

Reviewer: ```suggestion
    inferred key is represented as a `STRUCT`. You can access individual fields within

```

>     the `STRUCT` via [STRUCT dereference](../developer-guide/ksqldb-reference/operators.md#struct-dereference).

Reviewer: ```suggestion
    the `STRUCT` by using [STRUCT dereference](../developer-guide/ksqldb-reference/operators.md#struct-dereference).

```
